scripts/jenkins/jobs-ibs/cloud-mkcloud4-gating-trigger.yml

```diff
@@ -0,0 +1,47 @@
+- job:
+    name: 'cloud-mkcloud4-gating-trigger'
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H 6 */2 * *'
+      - pollurl:
+          cron: 'H/2 * * * *'
+          polling-node: cloud-trigger
+          urls:
+            - url: 'http://clouddata.cloud.suse.de/cgi-bin/grep/download.suse.de/ibs/Devel:/Cloud:/4:/Staging/images/iso/?regexp=x86_64'
+              check-content:
+                - simple: true
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          predefined-parameters: |
+            cloudsource=develcloud4
+            TESTHEAD=1
+            nodenumber=2
+            mkcloudtarget=plain crowbarbackup crowbarrestore testsetup
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          predefined-parameters: |
+            cloudsource=develcloud4
+            TESTHEAD=1
+            nodenumber=3
+            networkingplugin=linuxbridge
+
+    publishers:
+      - trigger-parameterized-builds:
+        - project:
+          - openstack-submit-project
+        condition: SUCCESS
+        predefined-parameters: |
+          project=Devel:Cloud:4
+          subproject=Staging
+          packagefilter=crowbar python openstack patterns release-notes-suse-openstack-cloud suse-cloud-upgrade
+          starttime=1999955453
```

Comment:
The starttime is used to not submit packages from Staging that have been changed since the test of them started. It must be overriden by the job triggering this one.